src/components/dev-hub/community-champion-form.js

```diff
@@ -175,7 +199,11 @@ const Form = React.memo(({ setSuccess, success }) => {
                 variant="default"
             >
                 {experienceOptions.map((option, index) => (
-                    <Radio key={index} value={option}>
+                    <Radio
+                        checked={experience == option}
+                        key={index}
```

Comment:
```suggestion
                        key={option}
```

Using `index` for key is typically an anti-pattern in React actually so let's avoid any potential weird bugs by swapping to another unique value here please!

https://robinpokorny.com/blog/index-as-a-key-is-an-anti-pattern/